src/ui/mainwindow.cc

```diff
@@ -3401,7 +3397,7 @@ void MainWindow::on_saveArticle_triggered()
     QWebEnginePage * page = view->page();
 
     // Connect the printFinished signal to handle operations after printing is complete
-    connect( page, &QWebEnginePage::pdfPrintingFinished, [ = ]( const QString & filePath, bool success ) {
+    connect( page, &QWebEnginePage::pdfPrintingFinished, page, [ = ]( const QString & filePath, bool success ) {
```

Comment:
If the page is gone, this slot should be gone too. I think this is the original semantics.